internal/controller/datadogagent/feature/otlp/feature.go

```diff
@@ -112,8 +112,10 @@ func (f *otlpFeature) Configure(dda *v2alpha1.DatadogAgent) (reqComp feature.Req
 // ManageDependencies allows a feature to manage its dependencies.
 // Feature's dependencies should be added in the store.
 func (f *otlpFeature) ManageDependencies(managers feature.ResourceManagers, components feature.RequiredComponents) error {
+	platformInfo := managers.Store().GetPlatformInfo()
+
```

Comment:
```suggestion
```
You can also set `versionInfo := platformInfo.GetVersionInfo()` here, and then use `versionInfo` for the two instances below